src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -653,6 +653,32 @@ public boolean isPayToScriptHash() {
     /**
      * Returns whether this script matches the format used for multisig outputs: [n] [keys...] [m] CHECKMULTISIG
      */
+    public boolean isSentToStandardMultiSig() {
+        if (chunks.size() < 4) return false;
+        ScriptChunk chunk = chunks.get(chunks.size() - 1);
+        // Must end in OP_CHECKMULTISIG[VERIFY].
+        if (!chunk.isOpCode()) return false;
+        if (!(chunk.equalsOpCode(OP_CHECKMULTISIG) || chunk.equalsOpCode(OP_CHECKMULTISIGVERIFY))) return false;
```

Comment:
This one looks confusing, negating an OR. Makes it hard to follow
```
if (!chunk.equalsOpCode(OP_CHECKMULTISIG) && !chunk.equalsOpCode(OP_CHECKMULTISIGVERIFY)) return false;
```
I think that one is clearer, what do you think?

Reply:
agree!